README.md

```diff
@@ -1,6 +1,7 @@
 # jv-salary-info
 
-Implement method `getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)`. It should calculate the salary for employees. As input, you receive two arrays and two dates in String format. 
+Implement method `getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)`. It should calculate the salary for employees. 
```

Comment:
Please don`t change readme

Reply:
not fixed, we should make it as it was before